lib/Doctrine/DBAL/Migrations/Migration.php

```diff
@@ -181,7 +181,7 @@ public function migrate($to = null, $dryRun = false, $timeAllQueries = false, ca
         $this->outputWriter->write("\n  <comment>------------------------</comment>\n");
         $this->outputWriter->write(sprintf("  <info>++</info> finished in %ss", $time));
         $this->outputWriter->write(sprintf("  <info>++</info> %s migrations executed", count($migrationsToExecute)));
-        $this->outputWriter->write(sprintf("  <info>++</info> %s sql queries", count($sql, 1) - count($sql)));
+        $this->outputWriter->write(sprintf("  <info>++</info> %s sql queries", count($sql, \COUNT_RECURSIVE) - count($sql)));
```

Comment:
Please import it instead (`use const`) to respect our CS - which is still being applied in #590